webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -484,7 +488,7 @@ class WebhookCreatePage extends Component {
 
             <div className="row">
               <div className="help-icon" id="name-tooltip">
-                <TooltipIcon tooltipText="The display name for your webhook in this user interface.">
+                <TooltipIcon onClick={this.preventTooltipPost} tooltipText="The display name for your webhook in this user interface.">
```

Comment:
`TooltipIcon` is intended to provide a tooltip explaining the purpose of an icon. If you want a standalone icon to trigger a tooltip for other purposes see [`Tooltip`](http://react.carbondesignsystem.com/?path=/story/tooltip--only-custom-icon)

However, in most cases forms should use a combination of labelText + helperText to present any relevant information to the user.

Reply:
the reason behind using TooltipIcon is because its the one that offers hovering instead of clicking